packages/block-editor/src/components/block-alignment-control/ui.native.js

```diff
@@ -77,7 +77,7 @@ function BlockAlignmentUI( {
 					};
 				} ),
 				popoverProps: POPOVER_PROPS,
-				toggleProps: { describedBy: __( 'Change alignment' ) },
+				toggleProps: { description: __( 'Change alignment' ) },
```

Comment:
@WordPress/native-mobile and @dcalhoun , just a quick check that this change makes sense and doesn't introduce unintended consequences 🙏

Reply:
@ciampo  Thanks for coordinating

Reply:
This change looks good to me 👍